src/zcl/definition/cluster.ts

```diff
@@ -1912,6 +1912,7 @@ const Cluster: {
             viessmannCustom13: {ID: 0x404c, type: DataType.enum8, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
             viessmannCustom14: {ID: 0x404d, type: DataType.bitmap8, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
             viessmannCustom15: {ID: 0x404e, type: DataType.bitmap8, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
+            schneiderWiserSpecific: { ID: 0xe110, type: dataType_1.default.enum8, manufacturerCode: manufacturerCode_1.default.SCHNEIDER },
```

Comment:
This syntax is invalid, please take a look at the errors above.

Reply:
I know, but github won't let me edit again (unless I'm on a branch).

(I'm a git newbie)